site/write_ups/2023/25-10-23_writeup.md

```diff
@@ -0,0 +1,66 @@
+
+Terminology was not the only problematic aspect of how the different groups of participants were separated. Interview techniques were very different for the different groups; whilst “everyday mums” were put in focus groups of 10 people, “influencer mums” had individual interviews. This doesn’t seem like a fair comparison, as relational dynamics may have an effect on the way conversations happen and the answers people provide. There were noticeably fewer “influencer” mums compared to “everyday” mums, and we wondered why this was the case. The diversity of participants was not discussed, and we wondered how results would change if the paper accounted for diversity aspects including separated parents and dads. This hugely limits the scope of participants.
+
+For the participants that were included, we thought that more instigative questions could have been asked. It seemed that the interviews just cracked the can of the topic, and then left it there without delving very deep. There could have been more critical provocation, especially towards the end of the interviews. 
+
+This highlights a missed opportunity for intervention. Asking questions about privacy encourages parents to reflect – for perhaps the first time. Including external resources in interviews, such as explanations about regulation, could have been a great learning experience. An interesting follow up could have been to examine if such an intervention did affect the behaviour of participants.
+
+Future work could expand to different social media platforms, like Reddit for instance. The paper is four years old, and a lot has changed in the realm of social media. We wondered if the findings are outdated now, and if there are differences in how people consume social media. The author has since moved into other areas, so we couldn’t find any further studies.
+
+## Bonus Question: What change would you like to see on the basis of this piece? Who has the power to make that change?
+
+Currently, the minimum age to hold an account on social media is about 13. However, the system allows us to represent children much younger than that. We wondered if this is fair, as young children do not have autonomy over the images shared of them. Without a flat ban, which seems unrealistic, this is difficult to navigate. The line denoting where posting is going beyond sharing a photo with friends is unclear, and perhaps should be further discussed. 
+
+
+## Attendees
+
+- Noshin Mohamed, Quality Assurance in Children's Services 
+- Natalie Zelenka, Senior Research Fellow in Health Data science, @NatalieZelenka GitHub, 
+- Lucy Bowles, Data Scientist at Brandwatch 
+- Jessica Woodgate, PhD student at University of Bristol
```

Comment:
Should we put something like "and others" when we know there were quite a few other attendees?